### DISCUSSION TITLE
UTM on visionOS

### DISCUSSION THREAD

**Discussion by austintatiousness** [utmapp/UTM#6360] [UTM Help (iOS)]
**Describe the issue**
Most of the VMs offered on the website crash UTM SE when you try to boot them. This is from UTM SE offered on the Apple Vision Pro App Store**

**Configuration (required)**
* UTM Version: 
* OS Version: visionOS 1.1.2
* Device Model: First Generation
* Is it jailbroken (name jailbreak used)? 
* How did you install UTM? from the visionOS app Store

ALSO can anyone explain why it is mentioned  nowhere that UTM SE made it onto the visionOS App Store? I am finding that absolutely incredible but also why isn't it discussed?

**Comment by osy** [comment]
I'm not saying anything until they approve iOS too in case it was an accident...
